packages/backend/package.json

```diff
@@ -10,10 +10,10 @@
   "scripts": {
     "start": "node --trace-warnings dist/index.js",
     "build": "yarn generate && tsc -b",
-    "dev": "tsc --build && concurrently \"tsc --watch --preserveWatchOutput\" \"nodemon\" \"yarn generate --watch\"",
+    "dev": "tsc --build && concurrently 'tsc --watch --preserveWatchOutput' nodemon 'yarn generate # --watch'",
     "typecheck": "yarn build",
     "precommit": "yarn lint-staged",
-    "generate": "DEBUG=1 graphql-codegen --config=codegen.yml",
+    "generate": "DEBUG=1 graphql-codegen --config=codegen.yml && export OUT=src/lib/autogen/daohaus-sdk.ts && awk '!/MolochVersion = .molochVersion/' $OUT > $OUT.filtered && mv $OUT.filtered $OUT",
```

Comment:
Is the proper fix to migrate to the newer daohaus graph?

Reply:
Just a question, we don't have to upgrade now

Reply:
Yeah. I did the `awk` script before DAOHaus got back to us.

There's a ticket, #1529, to replace the endpoint.